### DISCUSSION TITLE
Is there a way to use PKCS#11 URI?

### DISCUSSION THREAD

**Discussion by Geobert** [parallaxsecond/rust-cryptoki#185] [Q&A] [answered]
Hi,

I’m still trying to get head around this crate and it seems that I can’t find how to use a PKCS11 URI to access a token. 

What am I missing here?

**Comment by wiktor-k** [comment, marked as answer]
There's no support for that here. You may try this crate: https://crates.io/crates/pkcs11-uri (not affliated with this project, but looks relevant).
